config.yaml

```diff
@@ -5,6 +5,13 @@ options:
   cluster-name:
     description: "Optional - Name of the MySQL InnoDB cluster"
     type: "string"
+  profile:
+   description: |
+      profile representing the scope of deployment, and used to be able to enable high-level
+      high-level customisation of sysconfigs, resource checks/allocation, warning levels, etc.
+      Allowed values are: “production” and “testing”.
+   type: string
+   default: production
```

Comment:
CC: @delgod , CC: mohamed, please approve the option name.

Reply:
Sure. This `yyp` from the spec doc though